src/components/policy/src/cache_manager.cc

```diff
@@ -1187,6 +1187,10 @@ bool CacheManager::IsPredataPolicy(const std::string& app_id) {
       pt_->policy_table.app_policies_section.apps[app_id];
 
   policy_table::Strings res;
+
+  sort(pre_data_app.groups.begin(), pre_data_app.groups.end());
```

Comment:
@orhan-mehmedov is it std:sort? 
Where did useing namespace used? We clarified that we should use it inly in functions scope. Please use std explicitly